### PR TITLE
Updating doc regarding app deployment

### PR DESCRIPTION
**Pull request by procrastinatio**
Check the [README.md](https://github.com/geoadmin/web-mapviewer/blob/bug_update_doc_regarding_deploy/README.md)

[Test link](https://sys-map.dev.bgdi.ch/preview/bug_update_doc_regarding_deploy/index.html)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,13 +144,13 @@ The file `secrets.yml` will tell `summon` which keys to get from `gopass`.
 | ----------------------------- | -------------------------------------------- |
 | `npm run start`               | Compiles and hot-reloads for development. Will serve the project under `http://localhost:8080` (or the next available port if `8080` is already used, see console output). |
 | `npm run build`               | Compiles all file without bundling and minification |
+| `npm run build:(dev|int|prod)`| Compiles all file for the according `mode` |
```
Comment (ltshb) [inline README.md:147]: Somehow this is not rendered correctly:
image

File: README.md [hunk 2]
```diff
@@ -161,12 +161,50 @@ The CI uses this file to ensure it will not stumble upon a minor version of a li
...
 
-The target bucket will be defined by the target you've specified (`npm run deploy:dev|int`).
+### Automatic deploy
 
+After every successful build, a version of `develop` and `master` are deployed
+automatically.
+
+| environment 	| hostname             	| path          	| branch       	|
+|-------------	|----------------------	|---------------	|--------------	|
+| PR          	| sys-map.dev.bgdi.ch  	| /<branch_name> 	| <bug-*/feat-*>|
```
Comment (ltshb) [inline README.md:175]: The path is `/preview/<branch_name>`

**Review by ltshb — COMMENTED** [review]
Nice thanks, just two minor issues.

File: README.md (hunk 2 above)
Comment (procrastinatio) [reply README.md]: OK. Merci

**Review by ltshb — APPROVED** [review]
👍🏼

**Pull request merged**